startup.sh

```diff
@@ -101,7 +101,7 @@ if ! is_container_created "$TF_DEVENV_CONTAINER_NAME"; then
     --name $TF_DEVENV_CONTAINER_NAME \
     -w /root ${options} \
     -e CONTRAIL_DEV_ENV=/root/tf-dev-env \
-    $volumes \
+    $volumes -i \
```

Comment:
Here is detached mode  and the call below there is a call of 'show_progress.sh' that follows the logs till the end of the autobuild and returns the result. Doesnt it work for you?

Reply:
container exists, right after the build is complete.
or more it doesn't start in the first place if AUTOBUILD is set 0